integration-tests/src/test/java/org/apache/druid/tests/indexer/ITAppenderatorDriverRealtimeIndexTaskTest.java

```diff
@@ -139,4 +139,10 @@ String getQueriesResource()
   {
     return REALTIME_QUERIES_RESOURCE;
   }
+
+  @Override
+  int getNumExpectedRowsIngested()
+  {
+    return 22;
```

Comment:
Where does 22 come from?

nit: make this a constant so we can add an explanation in the javadocs? Similar comment for ITRealtimeIndexTaskTest

Reply:
Sounds good. Added.